src/newlines.cpp

```diff
@@ -2223,6 +2249,41 @@ static void newlines_brace_pair(chunk_t *br_open)
             }
             chunk_flags_set(br_open, PCF_ONE_LINER);         // set the one liner flag if needed
             chunk_flags_set(br_close, PCF_ONE_LINER);
+
+            if (  options::code_width() > 0
+               && br_close->column > options::code_width())
```

Comment:
What happens with cases where the brace close is not the last element? Off the top of my head I can only think of classes/structs and lambdas where the braces are followed by a semi colon.